archesproject/arches
# 11591 cbyrd remove declarations.d.ts testing

### Types of changes


### Description of Change




TO TEST:

1. on dev/8.0.x, create a project
2. ensure the instance of Arches it's pointing to does not have a `tsconfig-paths.json` file
3. run `npm run vitest`
4. notice the error mentioned in #11591 
5. run updateproject, or create a new project when checked out into this branch
6. run `npm run vitest`
7. notice the error no longer appears





### Issues Solved

Closes #11591 

### Checklist

-   I targeted one of these branches:
    - [x] dev/8.0.x (under development): features, bugfixes not covered below
    - [ ] dev/7.6.x (main support): regressions, crashing bugs, security issues, major bugs in new features

#### Accessibility Checklist

[Developer Guide](https://arches.readthedocs.io/en/stable/developing/advanced/accessibility/)

| Topic            | Changed | Retested |
| ---------------- | ------- | -------- |
| Color contrast   |         |          |
| Form fields      |         |          |
| Headings         |         |          |
| Links            |         |          |
| Keyboard         |         |          |
| Responsive Design|         |          |
| HTML validation  |         |          |
| Screen reader    |         |          |


#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @ 

### Further comments
